recipes/pyode/meta.yaml

```diff
@@ -0,0 +1,63 @@
+# defining package and version
+{% set name = "pyode" %}
+{% set version = "0.16" %}
+{% set soversion = "0.16.0" %}
```

Comment:
Does this need to be a separate variable? Or can `{{ version }}.0` be used instead?

Reply:
don't know the plans, I would like to keep these uncoupled.